app/Http/Controllers/ProspectActivityController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Filters\ContactCustomerFilter;
+use App\ProspectActivity;
+use App\Repositories\ContactCustomerRepository;
+use App\Repositories\ProspectActivityRepository;
+use Carbon\Carbon;
+use Illuminate\Http\Request;
+
+class ProspectActivityController extends Controller
+{
+    //
+    private $contactCustomerRepo;
+    private $prospectActivityRepo;
+    public function __construct(ContactCustomerRepository $contactCustomerRepository, ProspectActivityRepository $prospectActivityRepository)
+    {
+        $this->contactCustomerRepo = $contactCustomerRepository;
+        $this->prospectActivityRepo = $prospectActivityRepository;
+    }
+    public function index()
+    {
+        return $this->sendSuccess(['prospect_activities' => $this->prospectActivityRepo->all() ], 'Prospect customers and notification count retrieved successfully');
+    }
+
+    public function show(ProspectActivity $prospect_activity)
+    {
+         $d =  $prospect_activity->load('prospectActivityType');
+
+        return $d;
```

Comment:
why do we have 2 return statements?

Reply:
Forgot to clean up the method, will fix that now